src/Mono.Android/Mono.Android.csproj

```diff
@@ -13,7 +13,8 @@
     <NoWarn>0618;0809;0108;0114;0465;8609;8610;8614;8617;8613;8764;8765;8766;8767;RS0041</NoWarn>
     <WarningsAsErrors>$(WarningsAsErrors);CS2002</WarningsAsErrors>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <DefineConstants>$(DefineConstants);JAVA_INTEROP</DefineConstants>
+    <DefineConstants Condition=" '$(AndroidRuntime)' == 'Mono' ">$(DefineConstants);JAVA_INTEROP;MONO</DefineConstants>
+    <DefineConstants Condition=" '$(AndroidRuntime)' == 'NativeAOT' ">$(DefineConstants);JAVA_INTEROP;NATIVEAOT</DefineConstants>
```

Comment:
I was debating the names here. Should it be `MONO_RUNTIME` and `NATIVEAOT_RUNTIME`?

Reply:
I'd prefer a `RUNTIME_` *prefix*, vs. a `_RUNTIME` suffix: `RUNTIME_MONOVM`, `RUNTIME_NATIVEAOT`, etc.

I'd *really* prefer the need for any such changes to `Mono.Android.dll`, if possible…